<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>110</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_Button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>73</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fish Buddies</string>
   </property>
  </widget>
  <widget class="QLabel" name="fishIcon">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fish-logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="regiUser_input">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="regiPass_input">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="userNam_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>192</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fName_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="regiFName_input">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lNam_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>370</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="regiLName_input">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>400</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-300</x>
     <y>-100</y>
     <width>3671</width>
     <height>1216</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width:100%;
height:100%;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/wind-wave-cartoon-water-waves-6e912dfb99f8506682a88de1e14af64b.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="regiWarning_label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>178</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>User name is already in used. </string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>submit_Button</zorder>
  <zorder>regiPass_input</zorder>
  <zorder>label_2</zorder>
  <zorder>password_label</zorder>
  <zorder>regiUser_input</zorder>
  <zorder>fishIcon</zorder>
  <zorder>userNam_label</zorder>
  <zorder>label</zorder>
  <zorder>fName_label</zorder>
  <zorder>regiFName_input</zorder>
  <zorder>lNam_label</zorder>
  <zorder>regiLName_input</zorder>
  <zorder>regiWarning_label</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
